src/ucp/wireup/select.c

```diff
@@ -804,15 +819,18 @@ static uint64_t ucp_ep_get_context_features(const ucp_ep_h ep)
     return ep->worker->context->config.features;
 }
 
-static double ucp_wireup_rma_score_func(ucp_context_h context,
+static double ucp_wireup_rma_score_func(ucp_worker_h worker,
+                                        ucp_rsc_index_t rsc_index,
```

Comment:
maybe instead of worker+rsc_index+iface_attr, we could pass just "ucp_worker_iface_t"?

Reply:
good catch, done